hudi-spark-datasource/hudi-spark-common/src/test/scala/org/apache/spark/sql/hive/TestHiveClientUtils.scala

```diff
@@ -45,7 +45,7 @@ class TestHiveClientUtils {
     hiveClient = spark.sharedState.externalCatalog.unwrapped.asInstanceOf[HiveExternalCatalog].client
   }
 
-  @Test
+  @Disabled("HUDI-9118")
```

Comment:
Should this test be moved to another test class so spark can be properly cleaned up?  Also, does `@TestInstance(Lifecycle.PER_CLASS)` or `TestHive` has any behavior that can cause issues in this test?